vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -17,8 +17,65 @@ using namespace facebook::react;
 
 namespace Microsoft::JSI {
 
-/*static*/ std::once_flag ChakraRuntime::s_runtimeVersionInitFlag;
-/*static*/ uint64_t ChakraRuntime::s_runtimeVersion = 0;
+namespace {
+
+class HostFunctionProxy {
+ public:
+  HostFunctionProxy(
+      facebook::jsi::HostFunctionType hostFunction,
+      ChakraRuntime &runtime)
+      : m_hostFunction(hostFunction), m_runtime(runtime) {}
+
+  inline const facebook::jsi::HostFunctionType &getHostFunction() const {
+    return m_hostFunction;
+  }
+
+  inline ChakraRuntime &getRuntime() const {
+    return m_runtime;
+  }
+
+ private:
+  const facebook::jsi::HostFunctionType m_hostFunction;
+  ChakraRuntime &m_runtime;
+};
+
+class ArgsConverterForCall {
+ public:
+  ArgsConverterForCall(
+      ChakraRuntime &rt,
```

Comment:
>rt [](start = 21, length = 2)

(nit) spell out please

Reply:
Fixed locally.

---
In reply to: [333115547](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333115547) [](ancestors = 333115547)